service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java

```diff
@@ -130,9 +130,9 @@ public UsersRecord createNewUser(
   public void addToBlackList(String signature) {
     Timestamp expirationTimestamp = Timestamp.from(Instant.now().plusMillis(msRefreshExpiration));
     db.insertInto(Tables.BLACKLISTED_REFRESHES)
-            .values(signature, expirationTimestamp)
-            .onDuplicateKeyIgnore()
-            .execute();
+        .values(signature, expirationTimestamp)
```

Comment:
Was this automatic from spotless:apply?

Reply:
Good catch. Double check by running `mvn spotless:apply` @vrushalitarte

Reply:
yes it was automatic